core/src/main/scala/org/apache/spark/internal/config/Deploy.scala

```diff
@@ -75,5 +75,10 @@ private[spark] object Deploy {
     .intConf
     .createWithDefault(Int.MaxValue)
 
+  val CORES_RESERVED_FOR_APPS = ConfigBuilder("spark.deploy.coresReservedForApps")
+    .version("2.4.6")
```

Comment:
Hi, @tooptoop4 . We don't backport a feature. 
The feasible next version is 3.1.0.